vuepress/docs/next/tutorials/compose/page-management.md

```diff
@@ -1,82 +1,98 @@
 
--   The title of the selected content item will now be displayed in the widget settings.
+The selected content item will now be displayed.
 
--   Press the **Save** button.
+![image](./img/publish_page5b.png)
 
--   Click on the **Preview** button to show the page preview with the updated settings.
+**8. Click the **Save** button.**
 
--   Click on the **Publish** button to publish the page.
+The following actions are now available:
 
--   Click on the **View published page** button to view the published page in your application.
+-   Click on the `Preview` button to show the page preview with the updated settings.
+-   Click on the `Publish` button to publish the page.
+-   Click on the `View published page` button to view the published page in your application.
+
+## Page Templates
 
-## Create a Page Template
 Page Templates provide the scaffolding of a Page and are constructed using two main elements:
 
-1. A `JSON configuration` field which lists the set of frames that can be used on a page. 
-   - Each item in the frames array represents a frame or slot in the page, characterized by the following values:
-      - `pos` - a zero-based position index (starts from zero). This value is used in
-    APIs to address a specific widget on the page.
-      - `descr` - the frame description displayed on Page Design view
-      - `mainFrame` - designates the primary frame in the Page Template
-      - `defaultWidget` - widget code for a default widget to use in this frame. Page Template developers can use this field to provide suggestions on common widgets, e.g. header and footer widgets.
-      - `sketch` - an object with 4 coordinates (x1,x2,y1,y2) to allow the developer to place the widgets in the Page preview. Sketch’s x and y values go from 0 to 11 (similar to columns in Bootstrap), so if you want to place a 2x2 frame at the top left corner of the page, the values would be `x1: 0, x2: 1 y1: 0 y2: 1`.
-2. A `Template` field which uses Freemarker code to setup the HTML page itself.
-   - To add a frame in a specific place of the page, add `<@wp.show frame=0 />`, where frame is the pos variable from the `JSON configuration`. `<#assign wp=JspTaglibs["/aps-core"]>` is required at the top of the template to setup the `wp` variable.
-   - Common code can be shared across pages by using [UX Fragments](./widgets-fragments.md#create-a-ux-fragment) and ```<@wp.fragment code="\<FRAGMENT\_CODE\>" escapeXml=false /\>```.
+- A `JSON configuration` field which lists the set of frames that can be used on a page. 
+- A `Template` field which uses Freemarker code to setup the HTML page itself.
+
+### JSON Configuration
+
+Each item in the `frames` array of the JSON configuration represents a frame or slot in the page, characterized by the following values:
+- `pos`: A zero-based position index (starts from zero). This value is used by APIs to address a specific widget on the page.
+- `descr`: The frame description displayed in the Page Design view.
```

Comment:
Should Page Design or Page Template be backticked? I'm running into this and am not sure

Reply:
no, i don't think so... we haven't adopted that convention and i don't think it's necessary; we already have a lot of backticking going on and to do so would dilute the significance of where it's really applicable. what do you think?